docs/pages/tutorial/eas/ios-development-build-for-devices.mdx

```diff
@@ -0,0 +1,229 @@
+---
+title: Create and run a cloud build for iOS device
+sidebar_title: iOS development build for devices
+description: Learn how to configure a development build for iOS devices using EAS Build.
+---
+
+import { Step } from '~/ui/components/Step';
+import { Terminal } from '~/ui/components/Snippet';
+import ImageSpotlight from '~/components/plugins/ImageSpotlight';
+import { Collapsible } from '~/ui/components/Collapsible';
+import { BoxLink } from '~/ui/components/BoxLink';
+import { BookOpen02Icon } from '@expo/styleguide-icons';
+
+In this chapter, let's create our development build for iOS devices with EAS Build.
+
+## Trigger a build
+
+Development builds for iOS devices are generated in the **.ipa** format, which is standard for iOS app installations.
+
+> **Note**: Apple Developer membership is required to create and install a development build on an iOS device.
```

Comment:
Can we put the simulator build instructions before the device build instructions? I think a lot of people will hold out on getting an Apple Developer account if they can so they'd want to skip this and only so the simulator build.

Reply:
I wonder if in general it's worth explaining build signing before diving into the registration? Especially the why you need an apple dev account.

Reply:
> Can we put the simulator build instructions before the device build instructions? I think a lot of people will hold out on getting an Apple Developer account if they can so they'd want to skip this and only so the simulator build.

That's a good point! I'll adjust the navigation accordingly to put the iOS simulator before iOS in the navigation structure.

Reply:
> I wonder if in general it's worth explaining build signing before diving into the registration? Especially the why you need an apple dev account.

I'm currently thinking of providing this context as: "Each build is signed with [necessary credentials](/app-signing/app-credentials/#ios) to ensure that the app is from a trusted source. **Apple Developer account membership is required because it provides access to these credentials for signing our app.** EAS Build simplifies this process by managing those credentials for us."

- It provides link to learn more about what those credentials are in the App credentials doc.
- Do you think it answers the question? Or is it still complicated?

Reply:
I think that's perfect: enough info to provide context, but not a massive detour from the lesson point.